plugins/atlas/credentials.go

```diff
@@ -0,0 +1,50 @@
+package atlas
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func Credentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name: credname.Credentials,
```

Comment:
Should we create another credname here for understandability? `Credentials` is kind of general so I was thinking `PrivateKeyPair` (a name for the public/private key pair) could have more significance here.

Reply:
My only concern is that `PrivateKeyPair` wouldn't be a relevant credential name when we introduce config file-based authentication in the future, because it uses a different type of authentication -- based on Joqim's PR description.

Reply:
Indeed, but would it not remain relevant for the existing use case of the env vars?

Reply:
I agree with shipping this for now. 👍🏼 I am keen to understand whether renaming the credential name in the future would be considered a breaking change though.

Reply:
Generally, yes, this would make for a breaking change.

However, it should not happen that we have to change that name here. In this specific case, when we introduce a new way of authentication, we'll likely want to create a second `CredentialType` altogether, one that is not a private key pair, and will be called `access token` from what I read.